proto/internal/temporal/server/api/persistence/v1/chasm.proto

```diff
@@ -100,5 +113,5 @@ message ChasmTaskInfo {
 
     // Opaque attached task data. May be nil. Usable by components, not the CHASM
     // framework itself.
-    bytes data = 3;
+    temporal.api.common.v1.DataBlob data = 3;
```

Comment:
We may have different encoding in the future so using a DataBlob here. 
However, like commented  above, we could store the `versioned_transition_offset` here and avoid double storing task data.